### PR TITLE
#91 admin menu order

### PR DESCRIPTION
**Pull request by thatbudakguy**
refs #91 

- delete customized admin template, revert to stock grappelli
- create `dashboard.py` class to govern admin menu & organize sections accordingly

### CODE REVIEW

**Comment by thatbudakguy** [comment]
would be good if someone could test the "administration" menu and the "previous changes" menu - I can't see them in dev, presumably because I'm using a CAS login and I'm not an admin?

File: mep/dashboard.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            ]),
+        ))
+
+        # Bibliography
+        self.children.append(modules.ModelList(
+            _('Bibliography'),
+            column=1,
+            collapsible=False,
+            models=([
+                'mep.books.models.Item',
```
[the remark this answers is not in the document]
Comment (thatbudakguy) [reply mep/dashboard.py]: From what I can tell, [grappelli's model](http://django-admin-tools.readthedocs.io/en/latest/dashboard.html#the-modellist-class) lets you:
- specify all the models in an app
- specify a list of some models
but maybe not both at once? I'll try and see what happens, but I don't see an option for it

File: mep/dashboard.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    def init_with_context(self, context):
+        site_name = get_admin_site_name(context)
+
+        # Accounts
+        self.children.append(modules.ModelList(
+            _('Library Accounts'),
+            column=1,
+            collapsible=False,
+            models=([
+                'mep.accounts.models.Account',
```
[the remark this answers is not in the document]
Comment (thatbudakguy) [reply mep/dashboard.py]: Looks like not:
```
TypeError at /admin/
expected str, bytes or os.PathLike object, not ModelBase
```

[the remark this answers is not in the document]
Comment (rlskoeser) [reply mep/dashboard.py]: @thatbudakguy too bad; thanks for checking.

File: mep/dashboard.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (rlskoeser) [reply mep/dashboard.py]: Ok.  If not, then we just need documentation so we don't forget to add things as the models change.

[the remark this answers is not in the document]
Comment (thatbudakguy) [reply mep/dashboard.py]: looks like it's one or the other.

**Comment by rlskoeser** [comment]
Forgot to say in my review, but I did test the branch and it looks and works great.

**Pull request merged**